NuGet.Config

```diff
@@ -2,6 +2,13 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 
 <configuration>
+  <packageRestore>
+    <!-- Currently, the repository's version of NuGet.exe and Visual Studio's version
+         fight over the format of project.lock.json because the one in the respository
+         is newer. To prevent that, turn off package restore. -->
```

Comment:
Given we can't restore cross platform, what would be your guidance instead of checking them in?

If you aren't supposed to check them in, why did you put lock in the name? Every other dependency system uses "lock" == "lock dependencies".